docs/src/pages/components/Link/WebLinksPage.js

```diff
@@ -64,6 +64,19 @@ const LinkPage = () => (
     readme={linkReadme}
     examples={components}
     packageName="bpk-component-link"
+    usageTable={{
+      dos: [
+        'Each link on a page should clearly describe where it will take you if you choose to follow it. E.g. "Skyscanner privacy policy."',
+        'If a link contains an element such as an icon you should ensure it has suitable alt text for assistive technologies.',
+        'If links open in a new window, users of assitive technologies should be informed of this by applying the appropriate aria-label. E.g. The aria-label could be "opens in a new tab".',
```

Comment:
For these two do we need to make it clearer that the aria label should include the link text + the extra info? Otherwise someone might make a screen-reader just read "opens in a new tab" 🤔

Reply:
Yeah for sure! Wasn't 100% which order it would read it out so if it does the aria label first then the actual content, then totally makes sense to include both - will update this now 😄